client/src/common/SnackbarProvider.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState, useEffect, createContext } from "react";
+import ErrorSnackBar from "./ErrorSnackbar";
+import SuccessSnackBar from "./SuccessSnackbar";
+
+export const SnackbarContext = createContext();
+
+const SnackbarProvider = ({ children }) => {
+  //Snackbar Error State
+  const [isErrorSnackbarOpen, setIsErrorSnackbarOpen] = useState(false);
+  const [errorMessage, setErrorMessage] = useState("");
+
+  //Snackbar Success State
+  const [isSuccessSnackbarOpen, setIsSuccessSnackbarOpen] = useState(false);
+  const [successMessage, setSuccessMessage] = useState("");
```

Comment:
If we use types instead, we could also reduce the amount of state variables we have in this snackbar provider

Reply:
I removed some state variable to use a generic state for all types of snackbar